scripts/prepro_feats.py

```diff
@@ -50,6 +46,13 @@
 from misc.resnet_utils import myResnet
 import misc.resnet as resnet
 
+
+def append_array(f, f_toc, name, array):
+  np.save(f, array)
+  offset = f.tell()
+  f_toc.write("{} {}\n".format(name, offset))
+
```

Comment:
Shouldn't you set the offset before np.save?

Reply:
Good call